services/grizzly/setup.sh

```diff
@@ -30,6 +30,14 @@ EDIT=1 DESTDIR=/src ./fuzzmanager.sh
 ./gcov-9.sh
 ./taskcluster.sh
 
+# install mesa PPA
+# see https://bugzilla.mozilla.org/show_bug.cgi?id=1835691 for details
+# The fix was released as https://docs.mesa3d.org/relnotes/23.1.6.html
+# The fix should be included in Ubuntu 22.04.4 and 23.10 and this can be removed.
+sudo add-apt-repository -y ppa:kisak/kisak-mesa
```

Comment:
`add-apt-repository` is installed by `software-properties-common`, which isn't installed until line 115, so this should happen after that.

You also probably want to add `retry` and `-c main -c main/debug` to get the right dbgsyms